codes.md

```diff
@@ -583,7 +592,7 @@ further embellishment.
 These codes gather observations about the effect of communication on the
 mental model or depictions.
 
-KATY/ALEX: Is this a sub-code of depiction?
+TODO: KATY/ALEX: Is this a sub-code of depiction?
```

Comment:
I saw this -- what do you mean by "this"? Is "communication" a sub-code of depiction?

Reply:
Yeah I'm wondering whether it's a big enough code on its own or if it all ended up being depiction-based. It's hard because the line between mental model and depiction gets blurry. I'm fine leaving it on its own.

Reply:
Let's leave it on it's own for now